### PR TITLE
Add documentation for a method

### PR DESCRIPTION
**Pull request by kulshekhar**
Adds docs for the `getPropertyValue` method of the `CssStyleDeclaration` class in the `html` package

### CODE REVIEW

**Comment by mit-mit** [comment]
@terrylucas @jacob314 please review

**Comment by mit-mit** [comment]
@kevmoo can you see this gets reviewed, pls?

File: tools/dom/scripts/css_code_generator.py [hunk 1]
```diff
@@ -112,6 +112,12 @@ def GenerateCssTemplateFile():
     return style;
   }
 
+  /// Returns the value of the property if the provided *CSS* property
+  /// name is supported on this element and if the value is set. Otherwise
+  /// returns an empty string.
+  ///
+  /// Please note the property name uses camelCase, not-hyphens.
+  ///
```
Comment (kevmoo) [inline tools/dom/scripts/css_code_generator.py]: Unneeded extra line here...not a huge deal.

**Review by kevmoo — APPROVED** [review]
A tiny nit, but otherwise good.

**Comment by mit-mit** [comment]
Merging, thanks for contributing @kulshekhar !

**Comment by alan-knight** [comment]
@mit-mit @kevmoo Please note that this was not done correctly, and so didn't work. If you change the code generation scripts you have to run the go.sh script to regenerate the actual files.

**Pull request merged**